### PR TITLE
Created Links for Extension slot of Add Patient Button on Primary navbar

### PR DESCRIPTION
**Pull request by ujjwaltyagi355**
Hi, @FlorianRappl,
I have created these links for the extension slot for Add Patient button on the Primary Navbar.
For the issue [MF-464](https://issues.openmrs.org/browse/MF-464).
Do have a look and tell me if any changes are needed to be made.

### CODE REVIEW

**Comment by ujjwaltyagi355** [comment]
Hello, @FlorianRappl, 
I am unsure of what is causing the build failure.
Can you have a look?
Thank you.

**Comment by FlorianRappl** [comment]
Hi @ujjwaltyagi355 there is a type error:

> Error: src/index.ts(48,40): error TS2322: Type 'string' is not assignable to type 'Promise<{ default: ComponentType<unknown>; }>'.

Please correct the type.

In short, instead of getting the module loaded asynchronously, you just return the name of the module...

```js
        load : getAsyncLifecycle(() => ('./add-patient-link'),options),
```

should be

```js
        load : getAsyncLifecycle(() => import('./add-patient-link'),options),
```

Also please use [prettier](https://prettier.io/). Obviously, this code was not prettified. Thanks!

**Comment by ujjwaltyagi355** [comment]
@FlorianRappl,
Thanx for pointing that out.
Such a silly mistake I just made.

**Comment by ujjwaltyagi355** [comment]
Hello, @FlorianRappl,
Is this good?

File: src/add-patient-link.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { ConfigurableLink } from '@openmrs/esm-framework';
+import { Add20 } from '@carbon/icons-react';
+
+export default function Root() {
+  return (
+    <ConfigurableLink to="${openmrsSpaBase}/patient-registration">
```
Comment (FlorianRappl) [inline src/add-patient-link.tsx]: Should be wrapped in a `HeaderGlobalAction` with props:

```
aria-label="Add"
aria-labelledby="Add Patient"
name="AddPatientIcon"
```

**Review by FlorianRappl — CHANGES_REQUESTED** [review]

File: src/index.ts [hunk 2]
```diff
@@ -42,6 +42,11 @@ function setupOpenMRS() {
         slot: 'patient-photo',
         load: getAsyncLifecycle(() => import('./widgets/display-photo.component'), options),
       },
+      {
+        id: 'add-patient-primary-nav',
```
Comment (FlorianRappl) [inline src/index.ts]: add-patient-action

File: src/index.ts [hunk 3]
```diff
@@ -42,6 +42,11 @@ function setupOpenMRS() {
...
+        slot : 'add-patient-slot',
```
Comment (FlorianRappl) [inline src/index.ts]: top-nav-actions-slot

File: src/add-patient-link.tsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { ConfigurableLink } from '@openmrs/esm-framework';
+import { Add20 } from '@carbon/icons-react';
```
Comment (FlorianRappl) [inline src/add-patient-link.tsx]: Use the explicit module import, should be something like `@carbon/icons-react/es/components/add/20` or so

**Review by FlorianRappl — APPROVED** [review]
Great! :beers:

**Pull request merged**